source/MaterialXFormat/External/PugiXML/pugi.patch

```diff
@@ -0,0 +1,133 @@
+diff -crB v1.9/pugixml.cpp mtlx/pugixml.cpp
```

Comment:
This looks like a new file that has been accidentally included in the PugiXML folder?

Reply:
Actually, I added that file intentionally.  It captures the differences between the local pugiXML code, and the v1.9 release that was used as a base.

I'm not sure if the patch would ever cleanly merge on top of a newer release - but it was intended to make clearer the differences that the MaterialX project has made.

Reply:
Ah, I can understand the motivation for including the difference file directly, but I wonder if this actually increases our maintenance burden over time, as the difference file would need to be kept in sync with the actual delta between our code and the off-the-shelf version.

Maybe your text description at the bottom of the README is sufficient here, as a developer can always perform a difference between our code and the off-the-shelf code on their own?

Reply:
If you don't think it adds clarity - I'm happy to remove it - hopefully we can get back to a non-modified library at some point somewhat soon...

Reply:
Sounds good, and thanks!  Moving away from a modified PugiXML library seems like the right approach, and we'll just need to think through whether it should be accomplished through specification changes (i.e. requiring escape codes for angle brackets) or through refactoring of the codebase (i.e. moving all exceptional behavior from PugiXML to MaterialXFormat).